domain/consensus/processes/pruningmanager/pruningmanager.go

```diff
@@ -222,12 +222,16 @@ func (pm *pruningManager) nextPruningPointAndCandidateByBlockHash(stagingArea *m
 		}
 	}
 
-	ghostdagData, err := pm.ghostdagDataStore.Get(pm.databaseContext, stagingArea, blockHash, false)
+	currentPruningPoint, err := pm.pruningStore.PruningPoint(pm.databaseContext, stagingArea)
```

Comment:
Please use the same code that was used in https://github.com/kaspanet/kaspad/pull/2082